### ISSUE TITLE
It doesn't work if I change the default windows install location

### ISSUE THREAD

**Issue by Toadfield** [rocksdanister/lively#675]
**Describe the bug**
It doesn't work if I change the default windows install location.
When I then try to start lively wallpaper,then nothing happens.

**To Reproduce**
1. Open Regedit
2. Go to "Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion"
3. Edit ProgramFilesDir and ProgramFilesFir (x86) so that it uses a different folder (for example "C:\my programs\"
4. Restart your pc
5. Try to start lively wallpaper

**Expected behavior**
That Lively Wallpaper starts

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro 21H1

**Additional Information**
I need to set the default install location to something like that,so that the package manager chocolatey installs every program to there,so it isn't an option to disable that setting.

**Log file (Important)**
I can't give a log file cause it doesn't start.

**Comment by rocksdanister** [comment]
1. Are you installing the chocolatey package? if so then you will need to install x86 dotnet desktop runtime 3.1 manually.
2. Does it work if you install Lively to the required correct folder in the first place.
3. Windows Event Viewer should log the error.

**Comment by Toadfield** [comment]
> Are you installing the chocolatey package? if so then you will need to install x86 dotnet desktop runtime 3.1 manually.

I tried to install it with chocolatey package and with the normal installer,both doesn't work (I have dotnet desktop runtime 3.1 installed).
> Does it work if you install Lively to the required correct folder in the first place.

When I set in the lively installer that it should be installed to the custom windows default install path and then try to launch it,then it still doesn't work,if you meant that.
> Windows Event Viewer should log the error.

I can't find any source that is called "lively" or similar in the application tab in Event Viewer.

Btw other programs also don't install properly,so sorry if this issue isn't related to Lively.
